cpp/include/Ice/Proxy.h

```diff
@@ -586,7 +586,7 @@ namespace Ice
          * Obtains the facet for this proxy.
          * @return The facet for this proxy. If the proxy uses the default facet, the return value is the empty string.
          */
-        const std::string& ice_getFacet() const;
+        std::string ice_getFacet() const;
```

Comment:
Updated for consistency with the other Proxy APIs. None of them returns a const&.